src/test/java/rx/SingleTest.java

```diff
@@ -38,10 +38,13 @@
 import rx.functions.Action1;
 import rx.functions.Func1;
 import rx.functions.Func2;
+import rx.observables.BlockingSingle;
 import rx.observers.TestSubscriber;
 import rx.schedulers.Schedulers;
 import rx.subscriptions.Subscriptions;
 
+import static org.junit.Assert.*;
```

Comment:
Alright, well I'll stick with the wildcards for now. I can see your point about hiding used APIs (particularly for static imports). On the other hand, I've also had some gnarly merges cos of imports and import ordering - tho' generally for larger classes with more imports.